api/app/src/mappings/withings/body.ts

```diff
@@ -0,0 +1,78 @@
+import { Body } from "@metriport/api";
+
+import { Util } from "../../shared/util";
+import convert from "convert-units";
+
+import { PROVIDER_WITHINGS } from "../../shared/constants";
+import { WithingsMeasurements, WithingsMeasurementGrp } from "./models/measurements";
+
+export interface ResultsMeasurements {
+  [key: number]: number[];
+}
```

Comment:
NABD, but I'd choose `type` here instead of `interface`. They are very similar, but usually interfaces are more applicable to object-type data structures, whereas types are well suited to return types and parameters.

There's one feature of interfaces that I think is dangerous, which is declaration merging: declare the same interface twice, with diff contents, and the result is the merge of them.

https://www.typescriptlang.org/docs/handbook/2/everyday-types.html#differences-between-type-aliases-and-interfaces

Or, more detail: https://blog.logrocket.com/types-vs-interfaces-in-typescript/

But it's not a big deal, mostly to bring the conversation about the topic to our attention and eventually we converge to the same usage. 👍